README.md

```diff
@@ -1,218 +1,80 @@
+		return cloudant.getAllDbs().execute().getResult().stream().collect(Collectors.joining("\n"));
+	}
 
-public List<String> getAllDbs() {
-    return client.getAllDbs();
 }
-~~~
-
-## Related documentation
-* [Cloudant library for Java](https://github.com/cloudant/java-cloudant/)
-* [Cloudant documentation](https://console.bluemix.net/docs/services/Cloudant/cloudant.html#overview)
-* [Cloudant Learning Center](https://developer.ibm.com/clouddataservices/cloudant-learning-center/)
-* [Spring Boot documentation](https://projects.spring.io/spring-boot/)
-* [Spring Framework documentation](https://projects.spring.io/spring-framework/)
-
-# Development
-
-For information about contributing, building, and running tests see the [CONTRIBUTING.md](CONTRIBUTING.md).
+```
 
-### Using in Other Projects
+Run application:
 
-The preferred approach for using cloudant-spring in other projects is to use the Gradle or Maven dependency as described above.
+```shell
+./gradlew bootRun
+```
 
-### License
+REST request:
+```shell
+tomblench:cloudant-java-sdk-spring-sample$ curl http://localhost:8080/all_dbs
+kafka-test-db-04178485-8f15-4c3b-9138-d6ed68edb1d4
+kafka-test-db-07b6c52a-e2e9-4da5-b641-6233367f8af9
+kafka-test-db-0810edcf-fff6-4093-af01-f081fda77e74
+kafka-test-db-08d3cef8-50db-42a5-af9f-991493cbce50
+kafka-test-db-0c8255b0-ee10-4d4c-b104-def796305ebe
+...
+```
 
-Copyright © 2017, 2018 IBM Corp. All rights reserved.
+## Properties metadata
 
-Licensed under the apache license, version 2.0 (the "license"); you may not use this file except in compliance with the license.  you may obtain a copy of the license at
+The jar is built with `spring-configuration-metadata.json` which gives extra properties metadata and hints
+when editing a properties file in some IDEs. See screenshots below for examples:
 
-    http://www.apache.org/licenses/LICENSE-2.0.html
+![](Screenshot%20from%202022-06-29%2012-26-27.png)
 
-Unless required by applicable law or agreed to in writing, software distributed under the license is distributed on an "as is" basis, without warranties or conditions of any kind, either express or implied. See the license for the specific language governing permissions and limitations under the license.
+![](Screenshot%20from%202022-06-29%2012-27-38.png)
 
-### Issues
+## Issues
 
-Before opening a new issue please consider the following:
-* Please try to reproduce the issue using the latest version.
-* Please check the [existing issues](https://github.com/cloudant/cloudant-spring/issues)
-to see if the problem has already been reported. Note that the default search
-includes only open issues, but it may already have been closed.
-* When opening a new issue [here in github](../../issues) please complete the template fully.
+Currently the starter project only provides `spring-boot-starter` as a dependency, but for most users (as in the example
+above), the primary use case will be a web app, so should we provide `spring-boot-starter-web`?
```

Comment:
To answer the question here I think it is true that in many cases the web dependency will be used, but I don't think all use cases will need it so we should probably leave it up to the dev. For example, in a message oriented architectre a persistence microservice may well not need a web endpoint and it would be a shame to include it unnecessarily.

I think we should restore some information about raising issues to this section:
```suggestion
Before opening a new issue please consider the following:
* Please try to reproduce the issue using the latest version.
* Please check the [existing issues](/issues)
to see if the problem has already been reported. Note that the default search
includes only open issues, but it may already have been closed.
* When opening a new issue [here in github](/issues) please complete the template fully.
```